test/index.test.js

```diff
@@ -536,4 +536,233 @@ describe('Feathers Sequelize Service', () => {
       });
     });
   });
+
+  describe('ORM functionality with overidden getModel method', () => {
+    const EXPECTED_ATTRIBUTE_VALUE = 42;
+
+    function getExtraParams (additionalTopLevelParams, additionalSequelizeParams) {
+      additionalTopLevelParams = additionalTopLevelParams || {};
+      additionalSequelizeParams = additionalSequelizeParams || {};
+
+      return Object.assign({
+        sequelize: Object.assign({
+          expectedAttribute: EXPECTED_ATTRIBUTE_VALUE,
+          getModelCalls: { count: 0 }
```

Comment:
Note, keeping count in a separate object ensures that it is maintained as a reference rather than by value across any Object.assign() calls that service calls may do.